example_data/sample_bundle.json

```diff
@@ -108,45 +126,54 @@
             {
               "system": "http://snomed.info/sct",
               "code": "122555007",
-              "display": "Venus blood specimen"
+              "display": "blood"
             }
           ]
         },
         "subject": {
           "type": "Patient",
-          "reference": "Patient/37176e84-d977-e993-3c49-d76fcfc6e625"
+          "reference": "Patient?_id=gosh-redcap-40388914"
         }
       },
       "request": {
-        "method": "POST",
-        "url": "Specimen"
+        "method": "PUT",
+        "url": "Specimen?_id=specimen-20RG-205G0047"
       }
     },
     {
-      "fullUrl": "Practitioner/d3447490-96fd-35d0-adf2-0806e5214606",
+      "fullUrl": "Practitioner/practitioner1-20RG-205G0047",
       "resource": {
         "resourceType": "Practitioner",
-        "id": "d3447490-96fd-35d0-adf2-0806e5214606",
+        "id": "practitioner1-20RG-205G0047",
+        "identifier": [
+          {
+            "value": "practitioner1-20RG-205G0047"
+          }
```

Comment:
we didn't have an identifier before (won't comment on second one), but is this giving us any extra value?

Reply:
In an attempt for me to write the data directly to the FHIR server (instead of a folder as I have always done) using https://pypi.org/project/fhirpy/ approach, I saw that the practitioner resources required an identifier, which I will share with you when we meet. That was why I included the identifier.

Reply:
We'll probably need the identifier to be defined by the data so a normalised string of the first name, last name and the role
something like this in python `f"{first_name}_{last_name}_auth".to_lower().replace(" ", "_")`